src/components/common/TextArea/TextArea.style.ts

```diff
@@ -0,0 +1,59 @@
+import { css, Theme } from '@emotion/react';
+import { flexGenerator } from '@styles/generator';
+
+export const textAreaContainerStyle = css`
+  ${flexGenerator('column', 'center', 'flex-start')}
+  gap: 0.5rem;
+  margin: 0 2rem;
+`;
+
+export const textAreaWrapperStyle = (theme: Theme, isError: boolean) => css`
+  ${flexGenerator('column', 'space-between', 'end')}
```

Comment:
```suggestion
  ${flexGenerator('column', 'space-between', 'flex-end')};
```

flex-start는 기본값이라 다른값을 줘도 그냥 원래대로 보일텐데, flex-end는 이렇게 잘못된 값 넘겨주면 스타일 원하는대로 적용 안되지 않나요...?

Reply:
왜 되는거죠?? start나 end만 적어도 적용이 되네요 ...
그래도 확실한 이름으로 수정해두겠습니다!!!!